pkg/runs/runsGet.go

```diff
@@ -37,10 +37,16 @@ func GetRuns(
 		runJson, err = GetRunsFromRestApi(runName, timeService, apiServerUrl)
 		if err == nil {
 			var outputText string
-			outputText, err = chosenFormatter.FormatRuns(runJson, apiServerUrl)
+			if chosenFormatter.GetName() != "summary" {
+				runJson, err = GetRunDetailsFromRasSearchRuns(runJson, apiServerUrl)
```

Comment:
Shouldn't the input and output variable be a runArray as it is not in a json format at this point?